.github/workflows/main.yml

```diff
@@ -54,7 +54,7 @@ jobs:
 
       - uses: actions/setup-java@v4
         with:
-          java-version: 16
+          java-version: 17
```

Comment:
Would be nice to upgrade to 21... 😏

But that will also require updating the Gradle version as well.